codespell_lib/_codespell.py

```diff
@@ -1081,29 +1081,29 @@ def main(*args: str) -> int:
             print(f"    {ifile}: {cfg_file}")
 
     if options.regex and options.write_changes:
+        parser.print_usage()
```

Comment:
This pattern now shows up many times. Probably worth adding a little `_raise_usage_error(parser, msg)` for example to make the code DRY

Reply:
I have refactored the code as requested.